docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+There are 2 GPG public keys required by the instance configuration, that you will need to 
+copy to the new Admin Workstation.
+
+To copy the *Submission Key*, download it from the *Source Interface* and import
+it locally using the following commands
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop/install_files/ansible-base
+ curl http://$(cat app-source-ths)/journalist-key > SecureDrop.asc
+ gpg --import SecureDrop.asc
+
+Validate that the inported key's fingerprint matches the one on your 
+SecureDrop install. You can do this by first running the command:
+
+.. code:: sh
+
+ gpg -k --fingerprint 
+
+Then, compare the value with the fingerprint advertised by your *Source Interface*,
+using the command:
+
+.. code:: sh
+
+ curl http://$(cat app-source-ths)/metadata
+
+To copy the OSSEC alert public key, first list available keys on the monitor server:
+
+.. code:: sh
+ 
+ ssh mon sudo gpg --homedir=/var/ossec/.gnupg  -k
+
+Look for the key corresponding to the destination email address for OSSEC alert. Then, import it locally using the following commands (substituting the appropriate email address for ``alerts@example.com``:
+
+.. code:: sh
+
+ ssh mon sudo gpg --homedir=/var/ossec/.gnupg --export --armor alerts@example.com > ossec.asc
+ gpg --import ossec.asc
+ 
+You will be prompted for the fingerprints for both keys during the next step. To view their fingerprints, use the command:
+
+.. code:: sh
+ 
+ gpg -k --fingerprint
+
+Retrieve OSSEC alert configuration details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In addition to the OSSEC alert GPG key you retrieved above, you'll also need 
+the following configuration information:
+
+ - SMTP server
+ - SMTP port
+ - SASL username
+ - SASL domain
+ - SASL password
+
+To  these, use the following command in the terminal:
+
+.. code:: sh
+
+ ssh mon sudo cat /etc/postfix/sasl_passwd
+
+This will return a line like:
+
+.. code:: sh
+ 
+ [smtp.gmail.com]:587 testossec@gmail.com:AwfulPassword
+
+In this example, ``smtp.gmail.com`` is the SMTP server, ``587`` is the SMTP port,
+``testossec`` is the SASL username, ``gmail.com`` is the SASL domain, and 
+``AwfulPassword`` is the SASL password.
+
+(Optional) Retrieve HTTPS certificate files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If your *Source Interface* was configured to use HTTPS, you will need to copy
+three related files from the *Application Server* to the *Admin Workstation*.
+
+To retrieve these files, use the commands:
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop/install_files/ansible-base
+   ssh app sudo tar -c -C /var/lib ssl/  | tar xvf - 
+
+These commands will create a directory named 
+``~/Persistent/securedrop/install_files/ansible-base/ssl``
+on the *Admin Workstation*, containing your instance's SSL certificate, 
+certificate key, and chain file. When prompted for the names of these files
+during the next step, you should specify them relative to the 
+``install_files/ansible-base directory``, ie as ``ssl/mydomain.crt``.
```

Comment:
- `directory` does not belong in pre-formatted string
- typo: `ie` (should be `i.e.`)

Reply:
fixed